filters/exclusions.txt

```diff
@@ -36,9 +36,10 @@ $popup,~third-party
 !
 !### Anti-adblock baits ###
 ! all filters
-/###banner_ad$/
-/^##\.adsbox$/
-/^##\.afs_ads$/
+/^(~.*)?##\.adsbygoogle$/
+/^(~.*)?###banner_ad$/
+/^(~.*)?##\.adsbox$/
+/^(~.*)?##\.afs_ads$/
```

Comment:
why changed?

Reply:
Can there be harm? Some filter may add `~foo.com##.afs_ads` in future.

Reply:
Ok.